### DISCUSSION TITLE
Lidarr Extended seems much slower than AMD. Is there an option to increase the concurrency like AMD?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [RandomNinjaAtk/docker-lidarr-extended#86] [Q&A] [answered]
I increased the concurrency in AMD to like 5 or higher I believe. With AMD it would download something like 40,000 tracks a day. With Lidarr Extended it's only doing around 5k. Is there a way to increase the speed or concurrency?

**Comment by RandomNinjaAtk** [comment, marked as answer]
If you want to speed it up, you should disable replaygain, and don't re-encode to a different format if your using those features. This should operate at a pretty similar speed to AMD when replaygain and transcoding is not enabled. However, searching was greatly improved over the AMD implementation by making it more thorough, which results in more processing time, especially if your using both clients.

Also because its pulling from many online sources, its best not to overwhelm the services....

**Reply by julianrinaldi** [reply]
Ah ok, I'll try that, but I think I had it enabled before as well. Thanks for the great resource!
